WordPress/gutenberg
# Cover block: Refactor controls

## What?
Extracts the Cover block Block Control and Inspector Control components into separate files, along with a couple of other components and utils. 

## Why?
The Edit component file is currently large and hard to work with and almost impossible to test. This structure should hopefully make it easier to work with long term as well as make it possible to write unit/integration level tests for some of the block control functionality.

## How?
Mostly as straight move of code into separate files and a couple of variable/method renames, should be no change to functionality

## Testing Instructions

- Check that all aspects of the Cover block work as expected
- Check that `npm run test-unit packages/block-library/src/cover/test/block-controls.js` sample unit test runs
